src/styles/_landing.scss

```diff
@@ -231,19 +233,40 @@
     }
     > ul {
       #cosmetic-avenue {
-        background: url('/assets/img/landing/cosmetic-avenue/background.jpg');
-        .images {
-          img {
-            width: 45vw;
-            min-height: 90px;
-            @media (max-width: 600px) {
-              height: 13.8vw;
+        z-index: 1;
+        > div {
+          z-index: 0;
+          > .background {
+            //background images with parallax effect
+            position: absolute;
+            top: 0;
+            background: url('/assets/img/landing/cosmetic-avenue/background-texture.png');
```

Comment:
If all you want to do is set the `background-image` then don't use the shorthand `background`, instead use `background-image`, more declarative, sets only 1 thing.